tools/Status/Status.py

```diff
@@ -224,6 +233,34 @@ def render_status(show_paths, show_unidentified, state_styles, **kwargs):
     if len(job_data) == 0:
         return
 
+    # Remove deleted jobs
+    if not show_deleted:
+        deleted_jobs = job_data[~job_data["WorkDir"].map(os.path.exists)]
+        job_data.drop(deleted_jobs.index, inplace=True)
+
+    # Keep only latest in a series of segments
+    job_data[["SegmentsDir", "SegmentId"]] = [
```

Comment:
Can we add a column that prints what segment we are on? I think that could be useful. Then we don't have to include it in the job name like SpEC does.

Reply:
I had that before but didn't find it particularly useful. I can add it if you want though. Not sure.

Reply:
For long runs I think it'd be useful. It's a very quick way to know how many times the simulation has restarted.

Reply:
Right but is it useful to know the number of restarts? I don't know. Remember we have limited space in the terminal, so adding a column means things like the job name get truncated further.

Reply:
I feel like it should be an option. Everybody is going to want to see different things, so maybe (for a later PR) we should take the fields to print as arguments so users can customize them in their spectre.yaml file

Reply:
Yea that would be nice